Code/Framework/AzCore/Tests/Math/ObbTests.cpp

```diff
@@ -6,16 +6,16 @@
  *
  */
 
-#include <AzCore/Math/Obb.h>
+#include <AZTestShared/Math/MathTestHelpers.h>
 #include <AzCore/Math/Aabb.h>
-#include <AzCore/Math/Vector3.h>
+#include <AzCore/Math/Obb.h>
 #include <AzCore/Math/Transform.h>
+#include <AzCore/Math/Vector3.h>
 #include <AzCore/UnitTest/TestTypes.h>
```

Comment:
Why move the MathTestHelpers header to the top?

There is some controversy about people wanting headers in alphabetical order.

Reply:
The formatter ran in this case it seems and I'll be more watchful in the future. In this case however, it's benign. I _do_ believe in the ideal that header inclusion order should _not_ matter personally, so I'd be interested in where that discussion leads in the future.